doc/_static/style.css

```
@@ -3,12 +3,12 @@ html {
 }
 
 body {
-    font-family: 'Open Sans', sans-serif;
+    font-family: 'Tahoma', 'Open Sans', sans-serif;
```

I think we want to fall back to `sans-serif`, not our old font choice.
```suggestion
    font-family: 'Tahoma', sans-serif;
```